tuxemon/event/actions/choice_monster.py

```diff
@@ -0,0 +1,71 @@
+# SPDX-License-Identifier: GPL-3.0
+# Copyright (c) 2014-2023 William Edwards <shadowapex@gmail.com>, Benjamin Bean <superman2k5@gmail.com>
+from __future__ import annotations
+
+import logging
+from dataclasses import dataclass
+from functools import partial
+from typing import final
+
+from tuxemon.event import get_npc
+from tuxemon.event.eventaction import EventAction
+from tuxemon.locale import T, replace_text
+from tuxemon.states.choice.choice_monster import ChoiceMonster
+
+logger = logging.getLogger(__name__)
+
+
+@final
+@dataclass
+class ChoiceMonsterAction(EventAction):
+    """
+    Ask the player to make a choice among monsters.
+
+    Script usage:
+        .. code-block::
+
+            choice_monster <choices>,<variable>
+
+    Script parameters:
+        choices: List of possible choices
+            (monster slugs eg: rockitten:apeoro),
+            separated by a colon ":".
+        variable: Variable to store the result of the choice.
+
+    """
+
+    name = "choice_monster"
+
+    choices: str
+    variable: str
+
+    def start(self) -> None:
+        def set_variable(var_value: str) -> None:
+            player.game_variables[self.variable] = var_value
+            self.session.client.pop_state()
+
+        # perform text substitutions
+        choices = replace_text(self.session, self.choices)
+        maybe_player = get_npc(self.session, "player")
+        assert maybe_player
+        player = maybe_player
```

Comment:
I don't think we need two variables for the player.
Let's just do the check on the fly.
```suggestion
        player = get_npc(self.session, "player")
        assert player
```

Reply:
True, pointless repetition